Flabbergasted-Flamingos/Bangazon-admin-app
# Jcadddepartments

# Description
Can now click on the add department button in the department list view and get redirected to new department form.  Also added the add new employee to the employee list view

## Number of Fixes
All of them

## Related Ticket(s)
Close #6 

## Steps to Test Solution

1. git fetch --all
2. git checkout JCadddepartments
3. delete migrations just incase
3. python manage.py makemigrations/migrate
4. Run server
5. go to http://localhost:8000/admin_app/employees/
6. See if the add employee btn works. Should redirect you to the add employee form
7. go to http://localhost:8000/admin_app/departments/
8. See if add department btn works. Should redirect you to the add department form

## Testing

[x] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[x] I certify that all existing tests pass
